microcosm_postgres/encryption/encryptor.py

```diff
@@ -0,0 +1,76 @@
+"""
+Implement application-layer encryption using the aws-encryption-sdk.
+
+"""
+from typing import Mapping, Sequence, Tuple, Union
+
+from aws_encryption_sdk import KMSMasterKeyProvider, encrypt, decrypt
+
+
+class KMSEncryptor:
+
+    def __init__(self, key_ids: Sequence[str]):
+        """
+        Initialize the encryptor using a sequence of master key ids (or aliases).
+
+        Encryption calls will use *all* of these master key ids as part of the encryption
+        envelope. For disaster recovery, it is considered a best practice to use multiple
+        master keys from different AWS regions.
+
+        """
+        # NB: this key provider will generate a boto session using the runtime environment
+        self.key_provider = KMSMasterKeyProvider(key_ids=key_ids)
+
+    def encrypt(self,
+                encryption_context_key: str,
```

Comment:
nit: I am not sure what the formatting here should be.  As is, or:
```
def encrypt(
    self,
    encryption_context_key: str,
    plaintext: str,
) -> Tuple[bytes, Sequence[str]]:
```

Reply:
So my `emacs` formatter doesn't like your suggestion, so I never do it that way. I totally get why we'd format this way, so I never complain when people do. I've just never figured out how to make emacs handle this well.